e2e/dbt_project.yml

```diff
@@ -0,0 +1,33 @@
+# Name your project! Project names should contain only lowercase characters
```

Comment:
Can you make this file slimmer? Look at https://github.com/fal-ai/fal/blob/main/tests/mock/dbt_project.yml